.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length = 88
+select = E,F,W
```

Comment:
Is this just to avoid picking up codes from flake8 plugins if someone happens to install them?

Reply:
I copied from `dependency-file-generator`. If you have a preferred source for a default flake8 configuration, I will replace it with whatever you prefer.

You are correct that `select = E,F,W` limits the flake8 codes to the base set, and will not check error codes from plugins.

Reply:
I'm fine leaving this list for now. We can update later.